inotify/adapters.py

```diff
@@ -140,12 +142,14 @@ def _get_event_names(self, event_type):
 
         return names
 
-    def _handle_inotify_event(self, wd, event_type):
+    def _handle_inotify_event(self, inotify_fd, event_type):
```

Comment:
I don't think we need to pass `inotify_fd` around; its available via `self. __inotify_fd`, and passing it around makes it seem like there's more than one when there isn't

Reply:
actually, I think the original variable name is correct. There are potentially many watch descriptors attached to the inotify, and those are what get returned here

i'll change it back to wd

Reply:
Really? I'm confused now; it looks like the file descriptor passed into here is coming from the `self.__poll.poll`, and the `poll` is only watching `self.__inotify_fd`.

Additionally, from what I understand of how the linux inotify works, the watch descriptors aren't even files; you can't read from them, they're just identifiers that you can match with the files you asked to be watched

Reply:
ok, yes, you are right.

Reply:
ok - starting to tidy up this code becomes something of a rabbit hole. i'm going to stop at making it clear which file_descriptor is passed to handle_inotify_event and leave it there